src/Tasks/Common/TaskToolBase.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    public abstract class ToolTaskBase : ToolTask
```

Comment:
>ToolTaskBase [](start = 26, length = 12)

The file name doesn't match the class name. TaskToolBase.cs should be ToolTaskBase.cs